Gruntfile.js

```diff
@@ -33,22 +33,14 @@ module.exports = function (grunt) {
                     {expand: true, cwd: "./node_modules/d3/", src: "**", dest: "./src/lib/d3/"},
                     // Flocking
                     {expand: true, cwd: "./node_modules/flocking/", src: "**", dest: "./src/lib/flocking/"},
-                    // Infusion
-                    {expand: true, cwd: "./node_modules/infusion/build", src: "**", dest: "./src/lib/infusion"},
+                    // Infusion distribution files
+                    {expand: true, cwd: "./node_modules/infusion/dist", src: "**", dest: "./src/lib/infusion/dist"},
```

Comment:
In the demo, 'infusion-all.js' is from the path of 'src/lib/infusion':
https://github.com/colinbdclark/chartAuthoring/blob/FLOE-497/demos/index.html#L11

This line is putting it to the '/dist' sub-directory. This will cause a broken demo. 

Also, shall only the './node_modules/infusion/dist/infusion-all.js' be copied?